layers/+emacs/org/README.org

```diff
@@ -548,6 +548,9 @@ To install [[https://github.com/awth13/org-appear][org-appear]], which toggles v
    '((org :variables org-enable-appear-support t)))
 #+END_SRC
 
+If you set =org-appear-trigger= to =manual= and your editing style is modal,
```

Comment:
modal? do you mean vim or hybrid?

Reply:
Yes. Would it be better to phrase it that way?

Reply:
yes, please change it.

Reply:
Will do when I get a sec.

Reply:
```suggestion
If you set =org-appear-trigger= to =manual= and your editing style is =vim= or =hybrid=,
```